jobfunnel/__main__.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python3
 """Main script.
 
-Scrapes data off several listings, pickles it, and applies search filters.
+Scrapes data off several listings, pickles it, and applies search filters. 
```

Comment:
Can you get rid of the whitespace additions? 

This is best done with an extension or setting for most text editors via “remove training whitespace”